addons/mrp/models/mrp_production.py

```diff
@@ -2546,6 +2546,8 @@ def filter_by_attributes(record, product=self.product_id):
             byproduct_values.append(move_byproduct_vals)
         self.move_finished_ids += self.env['stock.move'].create(byproduct_values)
 
+        if self.warehouse_id.pbm_type_id:
```

Comment:
```suggestion
        if self.warehouse_id.manufacture_steps in ('pbm', 'pbm_sam'):
```

I believe `pbm_type_id` is always set by default :wink: